frontend/src/pages/modelServing/screens/projects/kServeModal/KServeDeploymentModeDropdown.tsx

```diff
@@ -28,8 +28,33 @@ export const KServeDeploymentModeDropdown: React.FC<Props> = ({ isRaw, setIsRaw,
       fieldId="deployment-mode"
       isRequired
       labelHelp={
-        <Popover bodyContent="Deployment modes define which technology stack will be used to deploy a model, offering different levels of management and scalability.">
-          <Icon aria-label="Model server replicas info" role="button">
+        <Popover
+          bodyContent={
+            <>
+              <div>Deployment modes determine the technology used to deploy your model:</div>
+              <ul style={{ paddingLeft: '1.2em', margin: 0 }}>
+                <li>
+                  <strong>Advanced</strong>: Knative Serverless, autoscaling, some customization
+                  limits.
+                </li>
+                <li>
+                  <strong>Standard</strong>: Kubernetes resources, simpler setup, no autoscaling to
+                  zero.
```

Comment:
I would also strongly recommend running this by Em Turner, they are a technical writer, for the wording and if it's consistent with the dashboard

Reply:
cc @eturner24

Reply:
No problem. Let's wait for her response :)

Reply:
I'm not sure how much room we have in the UI, but we could try to mirror something like what we have in our [official documentation](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.19/html-single/serving_models/index#about-kserve-deployment-modes_serving-large-models).

Perhaps:
```suggestion
                <li>
                  <strong>Advanced</strong>: Uses Knative Serverless but requires some manual customization. Supports autoscaling.
                  limits.
                </li>
                <li>
                  <strong>Standard</strong>: Uses Kubernetes resources with fewer dependencies and a simpler setup. Does not support autoscaling.
```

Reply:
Based on that approach this is the new solution:
(I didn't commit the changes yet, waiting for your approval first)
![image](https://github.com/user-attachments/assets/b106659e-83e0-4150-a4cd-505b40cc6978)

Reply:
Actually one last thing, the indents look slightly inconsistent, could we add the bullet points? 
![image](https://github.com/user-attachments/assets/2a3737e8-4085-491f-ae2d-19f36f641830)

There is also this which is another option (adding a gap between the paragraphs)
![image](https://github.com/user-attachments/assets/9c2411bd-6b6b-4e2e-aeb7-db36b63ddcde)

Reply:
Sure! On it.

Reply:
I rebased and pushed the changes. The current solution looks like this:
![Screenshot From 2025-04-29 17-17-22](https://github.com/user-attachments/assets/49abe1ad-d149-43df-be04-b85cb6c95040)